src/js/services/apiservice.js

```diff
@@ -0,0 +1,23 @@
+
+
+
+
+export const RegisterUser = (username, surname, passsword, email, phoneNumber, membershipType)=>{
+
+        fetch('http://ereader.retrotest.co.za/api/Db', {
+    method: 'POST',
+    headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+        "usersId": 0,
+        "name": username,
+        "surname": surname,
+        "email": email,
+        "phoneNumber": phoneNumber,
+        "password": passsword,
+        "membershipType": membershipType,
```

Comment:
remove trailing comma

Reply:
check all these files for these trailing commas before closing parentheses

Reply:
They have been removed in all files, thank you for your feedback.